dbt/logger.py

```diff
@@ -1,5 +1,6 @@
 import dbt.compat
 import logging
+import logging.handlers
```

Comment:
It took me a while to figure out why this is failing intermittently for me - it's because in normal operation something else imports logging.handlers and modules are singletons so everyone accessing `logging` has access to `logging.handlers`.

However, in some test runs for whatever reason, `dbt.logger.initialize_logger` executes before whatever other module that's loading does `import logging.handlers`, and so it doesn't see `logging.handlers` and fails.